docs/faq.md

```diff
@@ -51,8 +51,9 @@ For more options see the next question.
 
 ### Can I add a portion of the edits I made to a file, similarly to `git add -p` or `hg commit -i`?
 
-At the moment the best options to partially add a file are: `jj split`,
-`jj amend -i` and `jj move -i`.
+You can use `jj commit -i` to create a commit with a portion of your edits. You
+can also use `jj split -i`, `jj amend -i`, and `jj move -i` to accomplish
+similar workflows.
```

Comment:
I see `jj commit -i` as a feature to smooth the transition from other VCS, so should we write it something like this:

> Since the changes are already in the working-copy commit, the equivalent to
> `git commit -p`/`hg commit -i` is to split the working-copy commit with `jj split -i`
> (or the practically identical `jj commit -i`).
> 
> For the equivalent of `git commit --amend -p`/`hg amend -i`, use `jj squash -i`.

I think we should change the `git add -p` to `git commit -p` in the heading too. Maybe we should even change the heading to:

> Can I create a new commit from only some of the changes in the working copy, like `git commit -p` or `hg commit -i`?

Reply:
> I think we should change the `git add -p` to `git commit -p` in the heading too.

I think `git add -p` is a bit more appropriate, as I rarely have seen people commit directly. Usually people add subsets of the changes to the index before finishing their commit.

Reply:
> > I think we should change the `git add -p` to `git commit -p` in the heading too.
> 
> I think `git add -p` is a bit more appropriate, as I rarely have seen people commit directly. Usually people add subsets of the changes to the index before finishing their commit.

That's probably true. Maybe like this then?

> Can I create a new commit from only some of the changes in the working copy, like `git add -p && git commit` or `hg commit -i`?

Reply:
I updated it, let me know what you think.